src/lib/daoHelpers.ts

```diff
@@ -0,0 +1,55 @@
+import { DAO, IDAOState, Reputation, Token } from "@daostack/client";
+import { getArc } from "arc";
+import BN = require("bn.js");
+
+// TODO: all this should really be in the client library, after new client is done i will move it there if its not already
```

Comment:
please open an issue in the client for that please.

Reply:
https://github.com/daostack/client/issues/447